pkgs/development/compilers/adoptopenjdk-bin/jdk-linux-base.nix

```
@@ -10,10 +10,22 @@ sourcePerArch:
 , zlib
 , xorg
 , libffi
+, makeWrapper
+, glib
+, cairo
+, cups
+, gtk3
+, enableGTK ? true
```

```suggestion
, enableGTK ? true, glib, cairo, cups, gtk3
```

```suggestion
, enableGTK ? true
, glib, cairo, cups, gtk3
```

Grouping the option together with the optionals in some form makes it easier to know which things belong  to it.

---

Updated.  I will squash the fixup commits later.